src/application/components/ClientNotFoundModal.tsx

```diff
@@ -31,19 +32,33 @@ export function ClientNotFoundModal({
     <Modal open={open} onClose={onClose} size="xl">
       <Modal.Header>
         <Modal.Title>Could not find client</Modal.Title>
-      </Modal.Header>
-      <Modal.Body>
-        <p>
+        <Modal.Description>
           An Apollo Client instance was not found, either because an Apollo
           Client instance was never created, or because it could not be
           discovered. This is most commonly fixed by setting the{" "}
           <ConnectToDevToolsOptionLink /> to <code>true</code>.
-        </p>
-        <p className="mt-4">
+        </Modal.Description>
+        <Modal.Description>
           If this was reached in error, please try one of the following
           suggestions or click the &quot;Retry&quot; button to try looking for
           the client again.
```

Comment:
I _think_ this is the flow we want to encourage? (Spacing might need to be fixed.)

```suggestion
          If you've already set <ConnectToDevToolsOptionLink /> to <code>true</code>, please try one of the following
          suggestions or click the <strong>Retry</strong> button to try looking for
          the client again.
```